Backends/CLX/keysyms-common.lisp

```diff
@@ -121,7 +123,7 @@
 ;;; This dictionary maps CLX keysym names to the power-of-two
 ;;; constants that the CLIM II specification requires.
 (defconstant +clim-modifiers+
-  '(((:meta-left :meta-right) #.+meta-key+)
+  '(((:meta-left :meta-right :alt-left :alt-right) #.+meta-key+)
```

Comment:
I agree that they should be treated like other modifiers. That said Alt and Meta are semantically different things, so alt-left and alt-right should resolve to +alt-key+ (that is - we should map to this key -- n.b such constant is already defined).

Reply:
Added a comment as discussed.